test/functional/rpc_invalid_address_message.py

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env python3
```

Comment:
Not sure why this is being removed

Reply:
This was removed because the linter was complaining about the file permissions and that was the suggestion. I never deliberately changed any perms or ran anything as sudo. 

This will be resolved